src/cbmc/cbmc_parse_options.cpp

```diff
@@ -74,7 +75,7 @@ Function: cbmc_parse_optionst::cbmc_parse_optionst
 cbmc_parse_optionst::cbmc_parse_optionst(int argc, const char **argv):
   parse_options_baset(CBMC_OPTIONS, argc, argv),
   xml_interfacet(cmdline),
-  language_uit(cmdline, ui_message_handler),
+  language_uit(cmdline, ui_message_handler, std::make_shared<optionst>()),
```

Comment:
Re: shared_ptr, I'm just using this so that there's no possibility that we try to deref a null ptr/ref to an optionst. Another possibility is for language_uit to have an optionst as a direct data member. I'm reluctant to store a ref or raw-ptr to an optionst because it's too easy for the pointed-to object to go out-of-scope before the language_uit instance.